stock_picking_send_by_mail/models/stock_picking.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Sandra Figueroa Varela>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv
+
+
+class StockPicking(osv.osv):
```

Comment:
You can inspire you by https://github.com/OCA/stock-logistics-workflow/pull/353/files#diff-602dd04aa39038b2474d280f13cc567f